README.md

```diff
@@ -80,6 +80,26 @@ Logger.addLogAdapter(new AndroidLogAdapter() {
 
 ### Save logs to the file
 //TODO: More information will be added later
+
+Add permission in your `AndroidManifest.xml`:
+
+```xml
+<uses-permission
+        android:name="android.permission.WRITE_EXTERNAL_STORAGE"
+        android:required="true" />
+<uses-permission
+    android:name="android.permission.READ_EXTERNAL_STORAGE"
+    android:required="true" />
+<uses-permission
+    android:name="android.permission.WRITE_INTERNAL_STORAGE"
```

Comment:
Default implementation only requires external permissions, I think that'd be nice to state that only.